tests/lms/config/resources_test.py

```diff
@@ -32,6 +32,9 @@ def test_hypothesis_config_includes_the_authority(self, lti_launch):
             lti_launch.hypothesis_config["services"][0]["authority"] == "TEST_AUTHORITY"
         )
 
+    def test_hypothesis_config_disables_share_links(self, lti_launch):
+        assert lti_launch.hypothesis_config["services"][0]["enableShareLinks"] == False
```

Comment:
I'm not sure that it makes much difference but I think PEP8 would tell you to use `is False` here (I'm surprised the linter doesn't complain)

Reply:
`is <bool literal>` is certainly more idiomatic in h's code. Will fix.